### PR TITLE
i18n(es): Translated `components.mdx` 

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Translated `components.mdx`

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: docs/src/content/docs/es/guides/components.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Puedes mostrar una interfaz con pestañas utilizando los componentes `<Tabs>` y `<TabItem>`. Cada `<TabItem>` debe tener una `label` que se mostrará a los usuarios.
+
+```mdx
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<Tabs>
+  <TabItem label="Stars">Sirius, Vega, Betelgeuse</TabItem>
+  <TabItem label="Moons">Io, Europa, Ganymede</TabItem>
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:45]: We can translate the tab labels like you did for the cards here:

```suggestion
  <TabItem label="Estrellas">Sirius, Vega, Betelgeuse</TabItem>
  <TabItem label="Lunas">Io, Europa, Ganymede</TabItem>
```

File: docs/src/content/docs/es/guides/components.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+</Tabs>
+```
+
+El código anterior genera las siguientes pestañas en la página:
+
+<Tabs>
+  <TabItem label="Stars">Sirius, Vega, Betelgeuse</TabItem>
+  <TabItem label="Moons">Io, Europa, Ganymede</TabItem>
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:53]:
```suggestion
  <TabItem label="Estrellas">Sirius, Vega, Betelgeuse</TabItem>
  <TabItem label="Lunas">Io, Europa, Ganymede</TabItem>
```

**Review by delucis — COMMENTED** [review]
Looks great — thank you @Waxer59! I spotted a couple of places in the code examples we could also translate:

File: docs/src/content/docs/es/guides/components.mdx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+---
+
+import SomeComponent from '../../components/SomeComponent.astro';
+import AnotherComponent from '../../components/AnotherComponent.astro';
+
+<SomeComponent prop="algo" />
+
+<AnotherComponent>
+  Los componentes también pueden contener **contenido anidado**.
+</AnotherComponent>
```
[the remark this answers is not in the document]
Comment (Waxer59) [reply docs/src/content/docs/es/guides/components.mdx:25]: As long as they are indicative examples that do not affect any functionality, it's perfect!

**Review by delucis — APPROVED** [review]
Perfect — another PR ready to merge. Thanks for the great work 🙏

**Pull request merged**